config/buildspec_zero_code_change.yml

```diff
@@ -8,10 +8,12 @@ env:
     run_pytest_pytorch: "disable"
     run_pytest_mxnet: "enable"
     run_pytest_tensorflow: "disable"
+    run_pytest_tensorflow2: "disable"
     run_pytest_xgboost: "disable"
     run_integration_pytest_pytorch: "disable"
     run_integration_pytest_mxnet: "enable"
     run_integration_pytest_tensorflow: "disable"
+    run_integration_pytest_tensorflow2: "disable"
```

Comment:
where is this variable used? What test is run if it is enabled?

Reply:
@anirudhacharya

Reply:
not used anywhere for now. but I think we will need it eventually. 
 deleting it for now will not cause any issues.